build.cmd

```diff
@@ -106,7 +106,21 @@ if "%1" == "" goto ArgsDone
 if /i "%1" == "-?"    goto Usage
 if /i "%1" == "-h"    goto Usage
 if /i "%1" == "-help" goto Usage
+if /i "%1" == "--help" goto Usage
```

Comment:
I put in --help because this is seems to be the very standard on UNIX and thus many commands use it (e.g. git uses it).   If there is one command where we should be super-forgiving, it should be help.